### PR TITLE
Permissions and Usages

### PR DESCRIPTION
**Pull request by binaryoverload**
- [x] Completed PR - Has all features or fixes that it was meant to and isn't half done
- [x] Follows the style guide - Style guide found [here](https://github.com/FlareBot/FlareBot/blob/master/CONTRIBUTING.md)
- [x] All changes have been tested

## Added
 - Usages Command to allow users to get usages for any command
 - Group command for permissions to list groups

## Changed
N/A

## Deprecated
N/A

## Removed
N/A

## Fixed
N/A

## Security
N/A

### CODE REVIEW

File: src/main/java/stream/flarebot/flarebot/FlareBot.java [hunk 1]
```diff
@@ -8,12 +8,7 @@
 import com.arsenarsen.lavaplayerbridge.libraries.LibraryFactory;
 import com.arsenarsen.lavaplayerbridge.player.Track;
 import com.arsenarsen.lavaplayerbridge.utils.JDAMultiShard;
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
-import com.google.gson.JsonArray;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
+import com.google.gson.*;
```
Comment (weeryan17) [inline src/main/java/stream/flarebot/flarebot/FlareBot.java]: Didn't we just remove these not that long ago?

Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/FlareBot.java]: Eh I dunno xDD

**Comment by binaryoverload** [comment]
@weeryan17 Happy now? xD

**Comment by weeryan17** [comment]
yes

File: src/main/java/stream/flarebot/flarebot/FlareBot.java [hunk 2]
```diff
@@ -101,7 +102,7 @@
 import stream.flarebot.flarebot.web.ApiFactory;
 
 import javax.net.ssl.HttpsURLConnection;
-import java.awt.Color;
+import java.awt.*;
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/FlareBot.java]: Disable this

File: src/main/java/stream/flarebot/flarebot/commands/secret/DisableCommandCommand.java [hunk 3]
```diff
@@ -81,13 +82,13 @@ public CommandType getType() {
 
     @Override
     public String[] getAliases() {
-        return new String[] {"fuckthat", "kill", "disablecmd"};
+        return new String[]{"fuckthat", "kill", "disablecmd"};
     }
 
     private String getEmote(boolean b) {
-                     // tick - enabled (true)
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/commands/secret/DisableCommandCommand.java]: Don't "fix" this, it isn't formatted good 😂

**Comment by binaryoverload** [comment]
@WalshyDev Better? xD

File: src/main/java/stream/flarebot/flarebot/FlareBot.java (hunk 2 above)
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/FlareBot.java]: *whistles*

File: src/main/java/stream/flarebot/flarebot/commands/moderation/FixCommand.java [hunk 4]
```diff
@@ -1,14 +1,21 @@
...
 import stream.flarebot.flarebot.FlareBot;
 import stream.flarebot.flarebot.commands.Command;
 import stream.flarebot.flarebot.commands.CommandType;
 import stream.flarebot.flarebot.objects.GuildWrapper;
 import stream.flarebot.flarebot.util.ConfirmUtil;
+import stream.flarebot.flarebot.util.GeneralUtils;
 import stream.flarebot.flarebot.util.MessageUtils;
+import stream.flarebot.flarebot.util.objects.RunnableWrapper;
 
+import java.awt.*;
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/commands/moderation/FixCommand.java]: *whistles*

File: src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java [hunk 5]
```diff
@@ -14,8 +14,9 @@
 import stream.flarebot.flarebot.util.GeneralUtils;
 import stream.flarebot.flarebot.util.MessageUtils;
 
-import java.awt.Color;
+import java.awt.*;
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java]: *whie

File: src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java [hunk 6]
```diff
@@ -7,24 +7,19 @@
...
-import net.dv8tion.jda.core.entities.User;
+import net.dv8tion.jda.core.entities.*;
 import org.apache.commons.lang3.StringUtils;
 import stream.flarebot.flarebot.FlareBot;
 import stream.flarebot.flarebot.commands.Command;
 import stream.flarebot.flarebot.commands.CommandType;
 import stream.flarebot.flarebot.objects.Report;
 
-import java.awt.Color;
+import java.awt.*;
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java]: ``

File: src/main/java/stream/flarebot/flarebot/FlareBot.java (hunk 2 above)
Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/FlareBot.java]: OML

**Comment by binaryoverload** [comment]
@WalshyDev Better?

**Pull request merged**